modular/gater/migrate_handler.go

```diff
@@ -101,6 +103,67 @@ func (g *GateModular) checkMigratePieceAuth(reqCtx *RequestContext, migrateGVGHe
 	return false, nil
 }
 
+// checkMigrateBucketQuotaAuth parse migrateBucketMsgHeader to GfSpBucketMigrationInfo and check if request has permission
+// only used by preMigrateBucketHandler and postMigrateBucketHandler
+func (g *GateModular) checkMigrateBucketQuotaAuth(reqCtx *RequestContext, migrateBucketMsgHeader string) (bool, *gfsptask.GfSpBucketMigrationInfo, error) {
+	allow, bucketMigrationInfo, sp, err := g.verifySignatureAndSP(reqCtx, migrateBucketMsgHeader)
+	if !allow {
+		return allow, bucketMigrationInfo, err
+	}
+
+	effect, err := g.baseApp.GfSpClient().VerifyMigrateGVGPermission(reqCtx.Context(), bucketMigrationInfo.GetBucketId(), 0 /*not used*/, sp.GetId())
+	if effect == nil || err != nil {
```

Comment:
dont think need to put` effect == nil || err != nil` together and do ` if err != nil` within the if branch. Can check 1 by 1 instead.